### DISCUSSION TITLE
怎么让ChatGPT不要那么话痨

### DISCUSSION THREAD

**Discussion by trah01** [PlexPt/awesome-chatgpt-prompts-zh#179] [Q&A]
每次解释一堆，真的很烦且浪费tokens，有相关的提prompts可以让它的回答更加精炼且不要解释那么多吗

**Comment by PlexPt** [comment]
让他简要回答



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2023年9月13日(周三) 晚上7:49
收件人: ***@***.***&gt;;
抄送: ***@***.***&gt;;
主题: [PlexPt/awesome-chatgpt-prompts-zh] 怎么让ChatGPT不要那么话痨 (Discussion #179)




 
每次解释一堆，真的很烦且浪费tokens，有相关的提prompts可以让它的回答更加精炼且不要解释那么多吗

**Reply by trah01** [reply]
无用啊，，有没啥面具，能够真的让他少些话，，

**Comment by umuwovu227-alt** [comment]
提前限制它的长度
